operator/apis/execution/v1/scan_types.go

```diff
@@ -101,6 +103,8 @@ type HookStatus struct {
 	HookName string    `json:"hookName"`
 	State    HookState `json:"state"`
 	JobName  string    `json:"jobName,omitempty"`
+	Priority int       `json:"priority"`
```

Comment:
Why not `uint8`?

Reply:
I think having this be a signed int is helpful as this allows you to go both over and under the default prio (0).